frontend/next.config.js

```diff
@@ -47,4 +62,17 @@ module.exports = withTM({
       locales: ['en-US'],
       defaultLocale: 'en-US',
    },
-});
+   ...(!SENTRY_AUTH_TOKEN && {
```

Comment:
TIL that you can splat false and nul, huh. Makes for a clean syntax.
![image](https://user-images.githubusercontent.com/26855/172453062-90949e0c-7ef6-4fd9-8df4-96583deacea7.png)